skimmer/skimmer_utils.py

```diff
@@ -54,16 +104,21 @@ def apply_met_filters_cut(events, met_filter_names):
     """MET filters cuts.
 
     Args:
-        events (ak.Array)
+        events (EventsFromAkArray)
+        cut_flow (dict[str, float])
         met_filter_names (list[str])
 
     Returns:
-        ak.Array
+        None
```

Comment:
Why changing this? It is returning an ak array still

Reply:
solved